contributor-docs/CONTRIBUTING.md

```diff
@@ -78,65 +78,41 @@ Additionally, every component should support [the `sx` prop](https://primer.styl
 Here's an example of a basic component written in the style of Primer Components:
 
 ```jsx
-import {TYPOGRAPHY, COMMON} from './constants'
-import sx from './sx'
+import sx, {SxProp} from './sx'
 
-const Component = styled.div`
+const Component =
+  styled.div <
+  SxProp >
```

Comment:
Is there a reason this is split across three lines?

Reply:
Oh strange! Prettier did that for me. I'll fix.